utils/api/contracts/community.ts

```diff
@@ -0,0 +1,21 @@
+import { initContract } from '@ts-rest/core';
+import { z } from 'zod';
+import { extendZodWithOpenApi } from '@anatine/zod-openapi';
+import { communitySchema, communityUpdateSchema } from '../schemas/community';
+
+extendZodWithOpenApi(z);
+
+const c = initContract();
+
+export const communityContract = c.router({
+	update: {
+		path: '/api/community',
+		method: 'PUT',
+		summary: 'Update a community',
+		description: 'Update a community',
+		body: communityUpdateSchema,
+		responses: {
+			200: communitySchema,
+		},
+	},
+});
```

Comment:
I have only added updating communities here, not creating or deleting. I don't see the need to be able to do that through the API, seems like a possible abuse vector.
I would add the GET endpoint here when I finish that.